message_board/settings.py

```diff
@@ -203,6 +203,42 @@
 
 django_heroku.settings(locals())
 
+# Debugging in heroku live
+LOGGING = {
```

Comment:
why not this one? https://docs.djangoproject.com/en/3.1/topics/logging/#examples

Reply:
it seems easier to understand